src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
The method should be non-static
```suggestion
    public Ball getRandomBall() {
```